cosmos-redis/Dockerfile

```diff
@@ -5,4 +5,6 @@ FROM ${COSMOS_REGISTRY}/redis:6.2
 RUN mkdir /config
 COPY ./config/* /config/
 
+EXPOSE 3680
```

Comment:
What is this for?

Reply:
Expose controls what is shown when you do "docker ps" or whatever.  Unfortunately you can't remove an earlier EXPOSE, so I just added 3680 to the container so both 3679 and 3680 would show.